Views/ContactClient/datatable.html.php

```diff
@@ -20,6 +20,6 @@
     var tableData = <?php echo json_encode($tableData); ?>;
 </script>
 <?php
-    echo $view['assets']->includeScript('plugins/MauticContactClientBundle/Assets/build/contactclient.min.js?v=2', 'contactclientOnLoad', 'contactclientOnLoad');
+    echo $view['assets']->includeScript('plugins/MauticContactClientBundle/Assets/build/contactclient.min.js?v=2.15.1', 'contactclientOnLoad', 'contactclientOnLoad');
```

Comment:
Is it possible to pull the version from a config so this doesn't have to be updated ?